docs/manage-data/batches-load-purge.md

```diff
@@ -0,0 +1,66 @@
+# Load and Purge Batches
+
+The load and purge batch explanations are copied from the
+[admin commands](/docs/advanced/admin-commands.md) documentation.
+
+- [Load Batch](#load-batch)
+- [Purge Batch](#purge-batch)
+- [Obtain Batches](#download-batches)
+
+If you are using docker, batches should be stored in `data/batches`.
+
+## Load Batch
+
+This command loads the metadata and pages associated with a batch into a
+database and search index. It may take up to several hours to complete,
+depending on the batch size and machine. If there is a chance of interruption
+to where the command is running (for example, on a remote machine), you may
+wish to preface the command with `nohup (command) >> nohup.out` in order to
+ensure the batch will be completed regardless of your connection.
+
+Ensure that your batch location is a directory containing a `data` directory.
+
+```bash
+manage.py load_batch /path/to/batch_name
+```
+With docker, a path is not needed if your batch is in `data/batches`:
+
+```bash
+docker-compose exec web /load_batch.sh batch_name 
+```
+
+## Purge Batch
+
+This command removes pages associated with a batch from the search index and
+database, but does not remove titles specific to this batch. It is effective
+even on batches which were only partially loaded.
+
+```bash
+manage.py purge_batch batch_name
+```
+
+With docker:
+
+```bash
+docker-compose exec web manage purge_batch batch_name
+```
+
+## Obtain Batches
+
+There are a number of ways to obtain batches if you find yourself short of
```

Comment:
Pretty sure I already mentioned this elsewhere, but this might make more sense being a link to the other place we document how to get batches.

Reply:
Actually this whole file is duplicated content - maybe it should just be a link out to the other pages, and treated as an "index" of sorts?

Reply:
I am okay with either treating this as the index page or the other places pointing to it. For now, since I started at the top of your comments, I had already started making this the definitive page. I probably prefer that, since loading batches is probably the most common action once an app is set up, so I think it's okay to have a whole document just for it.  Load and purge batches could otherwise go with their relevant management commands, but I feel like the obtain batches section would then be bereft.
I'm flexible if y'all would rather have this document split up.

Reply:
Nah, works for me